README.md

```diff
@@ -2,42 +2,95 @@
 
 ## Applications
 
-Applications are located in bin directory. Before running those, please make
-sure that required Python packages (in python directory) are available:
+Applications are located in bin directory.
+Before running those, please make sure that required Python packages (in python directory) are available:
 
+```bash
 python3.8 -c "from lsst.ts.cRIOpy.GUI import *"
+```
+
+The above command shall pass without error.
+
+### M1M3GUI
+
+GUI/EUI for M1M3 static supports.
+Allows to perform various M1M3 operations, including FA bump testing and manual mirror positioning.
+Requires [ts_m1m3support](https://github.com/lsst-ts/ts_m1m3support) CSC running.
+
+To run the GUI with the CSC in simulation mode under the docker container, you can build the docker image of CSC first by:
```

Comment:
Why? There is dockerfile just for ts_cRIOpy, which has conda command run. It is much easier to just run docker build . -t m1m3gui. Running simulator (or running it with the real M1M3) is different topic.

Reply:
Added a new section title to clarify this in commit: 535c3c1. Thanks!

Reply:
Well can you please change this to use only ts_cRIOpy code? ts_m1m3simulator shall not be used. So the instruction would be:
```
docker build . -t crio_gui
```

you can than skip conda install, as dockerfile did just that. We don't know how ts_m1m3support will evolve, if we will need to change its dockerfile. There is Dockerfile in ts_criopy which will be updated to allow one to run EUIs and VMSlogger.